test/test_python_dispatch.py

```diff
@@ -83,7 +83,7 @@ def emit(self, record):
         self.log_list.append(f'{fmt_rets} = {record.msg}{fmt_args}')
 
 def log_input(name: str, var: object):
-    logging.getLogger("LoggingTensor").info("input", (name,), (var,))
+    logging.getLogger("LoggingTensor").info("input {} {}".format(name, var))
```

Comment:
I tried this out locally and it doesn't seem to solve the pytest issue. Do you have other ideas on how to fix this?

Using `python test/test_python_dispatch` I am getting:
```
======================================================================
ERROR: test_basic (__main__.TestPythonDispatch)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/raid/rzou/pt/fix_tensor_ctor_grad/test/test_python_dispatch.py", line 109, in test_basic
    log_input("x", x)
  File "/raid/rzou/pt/fix_tensor_ctor_grad/test/test_python_dispatch.py", line 91, in log_input
    logging.getLogger("LoggingTensor").info("input {} {}".format(name, var))
  File "/raid/rzou/pt/fix_tensor_ctor_grad-env/lib/python3.9/logging/__init__.py", line 1446, in info
    self._log(INFO, msg, args, **kwargs)
  File "/raid/rzou/pt/fix_tensor_ctor_grad-env/lib/python3.9/logging/__init__.py", line 1589, in _log
    self.handle(record)
  File "/raid/rzou/pt/fix_tensor_ctor_grad-env/lib/python3.9/logging/__init__.py", line 1599, in handle
    self.callHandlers(record)
  File "/raid/rzou/pt/fix_tensor_ctor_grad-env/lib/python3.9/logging/__init__.py", line 1661, in callHandlers
    hdlr.handle(record)
  File "/raid/rzou/pt/fix_tensor_ctor_grad-env/lib/python3.9/logging/__init__.py", line 952, in handle
    self.emit(record)
  File "/raid/rzou/pt/fix_tensor_ctor_grad/test/test_python_dispatch.py", line 85, in emit
    fmt_args = "(" + ", ".join(self._fmt(a) for a in record.args[0]) + ")"
IndexError: tuple index out of range
```
Using `pytest test/test_dispatch.py` I am getting:
```
___________________________________________________ TestPythonDispatch.test_basic ___________________________
Traceback (most recent call last):
  File "/raid/rzou/pt/fix_tensor_ctor_grad/test/test_python_dispatch.py", line 109, in test_basic
    log_input("x", x)
  File "/raid/rzou/pt/fix_tensor_ctor_grad/test/test_python_dispatch.py", line 91, in log_input
    logging.getLogger("LoggingTensor").info("input {} {}".format(name, var))
  File "/raid/rzou/pt/fix_tensor_ctor_grad-env/lib/python3.9/logging/__init__.py", line 1446, in info
    self._log(INFO, msg, args, **kwargs)
  File "/raid/rzou/pt/fix_tensor_ctor_grad-env/lib/python3.9/logging/__init__.py", line 1589, in _log
    self.handle(record)
  File "/raid/rzou/pt/fix_tensor_ctor_grad-env/lib/python3.9/logging/__init__.py", line 1599, in handle
    self.callHandlers(record)
  File "/raid/rzou/pt/fix_tensor_ctor_grad-env/lib/python3.9/logging/__init__.py", line 1661, in callHandlers
    hdlr.handle(record)
  File "/raid/rzou/pt/fix_tensor_ctor_grad-env/lib/python3.9/logging/__init__.py", line 952, in handle
    self.emit(record)
  File "/raid/rzou/pt/fix_tensor_ctor_grad/test/test_python_dispatch.py", line 85, in emit
    fmt_args = "(" + ", ".join(self._fmt(a) for a in record.args[0]) + ")"
IndexError: tuple index out of range
```

Reply:
I am not able to fix this issue. if you have any suggestions please tell and I will write the code. 
Thankyou

Reply:
I have made a pull request kindly see to it please
It is solving the issue

Reply:
Need anything more?
Rather than my PR?